changelog/updates/2023-05-04-package-updates-sys-apps.md

```diff
@@ -0,0 +1,17 @@
+- coreutils ([9.1](https://git.savannah.gnu.org/cgit/coreutils.git/tree/NEWS?h=v9.1))
+- debianutils ([5.7](https://metadata.ftp-master.debian.org/changelogs//main/d/debianutils/debianutils_5.7-0.4_changelog))
```

Comment:
I'd drop it, it's not a part of developer container nor production image, and probably not that important part of SDK anyway.